src/System.Private.ServiceModel/tools/TestRootCertificateInstaller/CertificateInstaller.cs

```diff
@@ -1,137 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Wow -- nice!  We drop the entire need for this projects now that we can directly curl the REST endpoint!